src/aiida_quantumespresso/parsers/xspectra.py

```diff
@@ -87,6 +87,28 @@ def parse(self, **kwargs):
         xy_data.set_y(y_arrays, y_names, y_units)
 
         parsed_data, logs = parse_stdout_xspectra(filecontent=out_file, codename='XSpectra')
+
+        # Parse some additional info which the stdout does not reliably report
+        input_params = self.node.get_incoming().get_node_by_label('parameters').get_dict()
+
+        epsilon_vector = [
+            float(input_params['INPUT_XSPECTRA'][f'xepsilon({n})']) for n in [1, 2, 3]
```

Comment:
```suggestion
            float(parameters[f'xepsilon({n})']) for n in [1, 2, 3]
```

Reply:
Should maybe consider `parameters.get(f'xepsilon({n})', default)` here to prevent `KeyError` if the key is not defined. But don't know what a sensible default could be here.